.env.sample

```
@@ -1,6 +1,7 @@
 # Secrets — provide via the environment (never in lemongrass.toml).
 INFLUX_TELEMETRY_TOKEN=
 RACEMONITOR_TOKENS=
+LEMONGRASS_DB_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LEMONGRASS_DB_PASSWORD key should go before the RACEMONITOR_TOKENS key

(UnorderedKey)